src/sqlfluff/core/rules/base.py

```diff
@@ -668,6 +717,15 @@ def discard_unsafe_fixes(
                 lint_result.fixes = []
                 return
 
+        for fix in lint_result.fixes:
+            if fix.has_template_conflicts(templated_file):
+                linter_logger.info(
```

Comment:
Is `info` there right log level for this?

For a similar error we use `warning`:
https://github.com/sqlfluff/sqlfluff/blob/7fd452dbfb9ab8537b8ca6e92c9489ff04d6172b/src/sqlfluff/core/linter/linter.py#L329-L331

Reply:
I used `info()` because other checks in the same function use it. I think it's the correct level, because this is routine behavior: code is templated, and rules may try to do things that are legit in templated space, but when mapped back to raw space, the edits can't be applied. Users don't care about this as long as we behave correctly (i.e. disacarding the fixes that can't be applied).